judo-tatami-jsl-jsl2psm/src/main/epsilon/transformations/psm/modules/structure/transferAttribute.etl

```diff
@@ -272,3 +282,66 @@ rule CloneTransferDefaultValueAttributeForErrorTransferObjectType
         t.setId("(jsl/" + s.getId() + ")/CloneTransferDefaultValueAttributeForDefaultTransferObjectType");
         log.debug("Clone TransferAttribute Default Value (Field) for ErrorTransferObjectType: [" + t.name + "]");
 }
+
+
+
+////////////////////////////////////////////////////////////////////////////////////////////
+// Transfer object field
+////////////////////////////////////////////////////////////////////////////////////////////
+@abstract
+rule AbstractCreateTransferAttributeForTransferFieldDeclaration
+    transform s: JSL!TransferFieldDeclaration
+    to t: JUDOPSM!TransferAttribute {
+        guard: s.referenceType.isKindOf(JSL!PrimitiveDeclaration)
+        t.required = s.isRequired;
+        t.name = s.name;
+        t.dataType = s.referenceType.getPSMEquivalent();
+
+        //if (s.defaultExpression.isDefined()) {
+        //    t.defaultValue = s.defaultExpression.equivalent("CreateDefaultValueForPrimitiveEntityMember");
+        //}
```

Comment:
Code in comments?

Reply:
There wil be rules which can be processed soon - this ticket is unfinished, only some parts are implemented, and Its a reminder where can and how this values.